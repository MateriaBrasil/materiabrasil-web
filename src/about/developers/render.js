import React from 'react'

import Colors from '../../Colors'
import Session from '../Session'
import Members from '../Members'
import Title from '../Title'
import Subtitle from '../Subtitle'
import Paragraph from '../Paragraph'
import developersList from './developersList'

export default () => (
  <Session id="developers">
    <Title color>Desenvolvedores</Title>
    <Paragraph color>
      Para o desenvolvimento desses direcionadores a MateriaMundi contou com uma
      equipe de consultores especialistas em diversas áreas.
    </Paragraph>
    <Subtitle>Conheça o nosso time!</Subtitle>
    <Members list={developersList} color={Colors.black} />
  </Session>
)
