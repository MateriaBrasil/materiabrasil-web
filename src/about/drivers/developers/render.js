import React, { Fragment } from 'react'

import Members from '../../Members'
import Subtitle from '../../Subtitle'
import Paragraph from '../../Paragraph'
import membersList from './developersList'

export default () => {
  return (
    <Fragment>
      <Subtitle>Desenvolvedores</Subtitle>
      <Paragraph>
        Para o desenvolvimento desses direcionadores a MateriaMundi contou com
        uma equipe de consultores especialistas em diversas áreas.
      </Paragraph>
      <Paragraph>Conheça o nosso time!</Paragraph>
      <Members membersList={membersList} />
    </Fragment>
  )
}
