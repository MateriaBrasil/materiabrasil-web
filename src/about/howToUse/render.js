import React from 'react'

import Section from '../Section'
import Title from '../Title'
import Video from '../../Video'
import VideoStyle from './VideoStyle'

export default () => (
  <Section
    id="how_to_use"
    backgroundImage="url('https://s3.amazonaws.com/materiamundi-us/static/backgrounds/material-3.jpg')"
  >
    <Title>Como usar</Title>
    <VideoStyle>
      <Video />
    </VideoStyle>
  </Section>
)
