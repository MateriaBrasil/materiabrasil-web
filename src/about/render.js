import React, { Fragment } from 'react'
import Typography from '@material-ui/core/Typography'

import Grid from './Grid'
import NavBar from '../navBar/NavBar'

export default props => routeProps => (
  <Fragment>
    <NavBar {...props} isTransparent="true" />

    <Grid>
      <Typography
        variant="display2"
        color="inherit"
        className="text"
        style={{
          color: '#fff',
          marginTop: 70,
          display: 'inline-block',
          fontWeight: 300,
        }}
      >
        <h2>
          <strong>Objetivo</strong>
        </h2>
        O objetivo da MateriaMundi oferecer acesso à informação de qualidade e
        parâmetros para que seus usuários possam escolher alternativas
        comercialmente disponíveis e socioambiental responsáveis para os
        desafios do dia a dia, variando desde o que vestimos até como
        construímos nossas casas.
        <br />
        Através da criação de uma comunidade ativa a plataforma permite a
        criação de conhecimento compartilhado sobre ações práticas para uma
        forma de projetar, produzir e consumir mais responsável com a sociedade
        e o meio ambiente.
        <h2>
          <strong>Como usar</strong>
        </h2>
        <h2>
          <strong>Equipe</strong>
        </h2>
        <h2>
          <strong>Board</strong>
        </h2>
        <h2>
          <strong>Direcionadores</strong>
        </h2>
      </Typography>
    </Grid>
  </Fragment>
)
