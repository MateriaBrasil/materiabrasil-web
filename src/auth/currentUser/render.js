import React from 'react'

import Loading from '../../Loading'

export default ({ render }) => (currentUser, props) => {
  const { fetchingInfo, info, infoError } = props

  if (fetchingInfo || (!info && !infoError)) {
    return <Loading />
  }

  return render(info, props)
}
