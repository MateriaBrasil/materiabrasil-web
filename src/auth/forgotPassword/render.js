import React from 'react'

import ForgotPassword from './ForgotPassword'

export default props => routeProps => (
  <ForgotPassword {...props} {...routeProps} />
)
