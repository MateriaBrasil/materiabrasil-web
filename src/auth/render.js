import React from 'react'

import Router from './Router'

export default props => routeProps => {
  return <Router {...props} />
}
