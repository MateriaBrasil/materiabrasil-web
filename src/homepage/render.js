import React from 'react';

import Home from './Home';
import NavbarAndContent from '../NavBarAndContent';

export default props => routeProps => (
  <NavbarAndContent institutional {...props}>
    <Home {...props} />
  </NavbarAndContent>
);
