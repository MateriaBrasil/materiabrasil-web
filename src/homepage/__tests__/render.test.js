import React from 'react'
import renderer from 'react-test-renderer'

import render from '../render'

jest.mock('../../navBar/NavBar', () => props => <div {...props}>NavBar</div>)
jest.mock('../Home', () => props => <div {...props}>Home</div>)

it('renders correctly', () => {
  const props = {}
  const routeProps = {}
  const tree = renderer.create(render(props)(routeProps)).toJSON()
  expect(tree).toMatchSnapshot()
})
