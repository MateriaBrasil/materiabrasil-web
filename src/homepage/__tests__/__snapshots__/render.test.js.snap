// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    isInstitutionalPage="true"
>
    NavBar
</div>,
  <div
    className="MuiGrid-container-2"
    style={
        Object {
            "backgroundImage": "url(\\"https://picsum.photos/g/1920/900/?random\\")",
            "backgroundPosition": "center",
            "backgroundSize": "cover",
          }
    }
>
    <div
        className="MuiGrid-item-3 MuiGrid-grid-xs-12-42 MuiGrid-grid-sm-8-52 MuiGrid-grid-md-6-64 Component-grid-1"
        style={
            Object {
                "position": "relative",
              }
        }
    >
        <h3
            className="MuiTypography-root-99 MuiTypography-h3-113 MuiTypography-colorInherit-128 text"
            style={
                Object {
                    "color": "white",
                    "display": "inline-block",
                    "fontWeight": 300,
                    "padding": "0 20px",
                    "verticalAlign": "middle",
                  }
            }
        >
            Na 
            <span
                style={
                    Object {
                        "fontWeight": 400,
                      }
                }
            >
                Materia
            </span>
            Mundi, nós ajudamos indústrias e consumidores a implementar ações para uma economia circular aplicada.
        </h3>
    </div>
</div>,
  <div
    className="MuiGrid-container-2"
    style={
        Object {
            "backgroundImage": "url(\\"https://picsum.photos/g/1920/900/?random\\")",
            "backgroundPosition": "center",
            "backgroundSize": "cover",
          }
    }
>
    <div
        className="MuiGrid-item-3 MuiGrid-grid-xs-12-42 MuiGrid-grid-sm-8-52 MuiGrid-grid-md-6-64 Component-grid-1"
        style={
            Object {
                "paddingTop": "35%",
                "position": "relative",
              }
        }
    >
        <iframe
            allowFullScreen="true"
            frameBorder="0"
            height="100%"
            mozallowfullscreen="true"
            src="https://player.vimeo.com/video/140946149"
            style={
                Object {
                    "border": 0,
                    "height": "100%",
                    "left": 0,
                    "maxHeight": "540",
                    "maxWidth": "960",
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                  }
            }
            title="Vídeo de introdução ao Materia Brasil"
            webkitallowfullscreen="true"
            width="100%"
        />
    </div>
</div>,
  <div
    className="MuiGrid-container-2"
    style={
        Object {
            "backgroundImage": "url(\\"https://picsum.photos/g/1920/900/?random\\")",
            "backgroundPosition": "center",
            "backgroundSize": "cover",
          }
    }
>
    <div
        className="MuiGrid-item-3 MuiGrid-grid-xs-12-42 MuiGrid-grid-sm-8-52 MuiGrid-grid-md-6-64 Component-grid-1"
        style={
            Object {
                "position": "relative",
              }
        }
    >
        <div
            style={
                Object {
                    "textDecoration": "none",
                  }
            }
            to="/materials"
        >
            Link - 
            <button
                className="MuiButtonBase-root-160 MuiButton-root-134 MuiButton-contained-145 MuiButton-raised-148 MuiButton-sizeLarge-158"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                    Object {
                        "background": "#fff",
                        "borderRadius": 7,
                        "fontSize": 40,
                        "fontWeight": 100,
                        "marginBottom": 50,
                        "padding": "15px 50px",
                        "textTransform": "lowercase",
                      }
                }
                tabIndex="0"
                type="button"
            >
                <span
                    className="MuiButton-label-135"
                >
                    Explore
                </span>
                <span
                    className="MuiTouchRipple-root-163"
                />
            </button>
        </div>
        <p
            className="MuiTypography-root-99 MuiTypography-body1-108"
            style={
                Object {
                    "bottom": 20,
                    "color": "white",
                    "fontFamily": "roboto",
                    "padding": "0 20px",
                    "position": "absolute",
                    "textAlign": "center",
                    "width": "100%",
                  }
            }
        >
            Apoio: Instituto C&A; 
            <br />
            Parceiros: Good Tech Lab, Fab City Global, Materiom;
            <br />
            Membro: Sistema B Brasil e CE100 Global Ellen MacArthur Foundation.
        </p>
    </div>
</div>,
]
`;
